### DISCUSSION TITLE
Will   Buttons work as well?

### DISCUSSION THREAD

**Discussion by frankhuettner** [JuliaPluto/PlutoSliderServer.jl#27] [Q&A] [answered]
I followed Connor wonderful tutorial (https://cotangent.dev/how-to-publish-pluto-jl-notebooks-online-interactively/) and tried Pluto's sample file PlutoUI. It turns out that `Button` (to trigger recomputation of a cell) doesn't work. 

Out of curiosity: Is this intentional? Screening the MIT course, I also found no example of a Button being used.

Cheers!
Frank

**Comment by fonsp** [comment, marked as answer]
Hi @frankhuettner ! It was not intentional to break the Button, this is a result of our caching: https://www.youtube.com/watch?v=QZ3xlKm92tk The Button sends the same value every time, so the result is cached, and further clicks have no effect.

A fix would be https://github.com/fonsp/PlutoUI.jl/issues/38

**Reply by fonsp** [reply]
I just implemented this widget, so update to PlutoUI 0.7.16 and use `CounterButton` instead of `Button` to fix your issue :)
